db/migrate/20180830104245_add_creator_id_to_document.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCreatorIdToDocument < ActiveRecord::Migration[5.2]
+  def change
+    add_column :documents, :creator_id, :integer
```

Comment:
It might be better to do this through a [`add_reference`](https://apidock.com/rails/v4.0.2/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference) since we're referencing the user table, and will match the data type (which I believe is a bigint)